### PR TITLE
 nginx-storage pod with persistent disk

### PR DESCRIPTION
**Pull request by bartoszbetka**
#15

### CODE REVIEW

File: kubernetes/Makefile [hunk 1]
```diff
@@ -16,7 +16,9 @@ CLUSTER_SCRIPTS :=                                     \
 	build/secrets/django-admin-fixture.yaml            \
 	build/jobs/create-database.yml                     \
 	build/jobs/migrate-database.yml                    \
-	build/jobs/delete-database.yml
+	build/jobs/delete-database.yml                     \
+	build/config-maps/nginx-storage/default.conf       \
+	build/services/nginx-storage.yml
```
Comment (cameel) [inline kubernetes/Makefile]: Please keep similar files together. This should be next to the other files in `build/services/`.

File: kubernetes/services/nginx-storage.yml.j2 [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                command: ["/usr/sbin/nginx", "-s", "quit"]
+          volumeMounts:
+            - mountPath:  /srv/storage/
+              name:       concent-storage
+            - mountPath:  /etc/nginx/conf.d/
+              name:       nginx-storage-config
+      volumes:
+        - name: concent-storage
+          gcePersistentDisk:
+            pdName: {{ persistence_disk_name }}
```
Comment (cameel) [inline kubernetes/services/nginx-storage.yml.j2]: `persistence` > `persistent`.

Also, "persistent disk" is a very generic name and does not explain well how this disk is different from others persistent disks we use. For example every cluster node has a persistent disk for its files attached. Call it `concent_storage_disk_name`.

**Comment by cameel** [comment]
@bartoszbetka Please fix the issues listed above so that we can merge the branch.

**Comment by bartoszbetka** [comment]
Issues fixed.

**Comment by cameel** [comment]
Code merged. I made a few small tweaks myself:
- Rewritten the snippet in the README
- Your path in `nginx-proxy` was missing the final slash.
- Changed the order in which pods are created and deleted. `nginx-proxy` should always be created last and deleted first because it's the entry point to the cluster. Similar with `nginx-storage` (but it's not in a cluster of its own yet so it's after `nginx-proxy`). Then everything else.
- I saw that `delete-services.sh` still does not ignore errors from `delete-config-maps.sh` which it calls. It was supposed to be fixed by now.

**Pull request closed without merging**